system-variables.md

```diff
@@ -304,6 +304,14 @@ SET  GLOBAL tidb_distsql_scan_concurrency = 10;
 - 这个变量用来设置 scan 操作的并发度。
 - AP 类应用适合较大的值，TP 类应用适合较小的值。对于 AP 类应用，最大值建议不要超过所有 TiKV 节点的 CPU 核数。
 
+### tidb_dml_batch_size
```

Comment:
```suggestion
### `tidb_dml_batch_size`
```

Reply:
Done in https://github.com/pingcap/docs-cn/pull/4590/commits/5a0f3aecb7290361496a61b00527f3de4c2e7672